orchestrate-containers/AnsibleRoles/playbook/app_deploy.yaml

```diff
@@ -0,0 +1,5 @@
+- hosts: kube-master
```

Comment:
An alternative to separating the various plays, you could put them all in a single YAML file that describes your site config. The idea is, if a 3rd-party ran this playbook, the entire infra would be configured at one shot. After combining, It will look something like this:
```yaml
- name: Deploy app
  hosts:
    kube-master
  become: yes
  roles:
    - role: app_deploy
      tags: app_deploy

- name: Ceph Master setup
  hosts:
    kube-master
  become: yes
  roles:
    - role: ceph_master_setup
      tags: ceph_master_setup
```

Each play can execute multiple roles. If you wanted to run just one play, use the command `ansible-playbook -Ki host_deploy --tags ceph_master_setup 05-site-config.yml`. The next thing I want you to look at is tagging.

Reply:
Roger! I will go research on tagging for Ansible to better understand the function.